book/glossary.md

```diff
@@ -10,329 +10,511 @@ clear names*.
+Event loop
+
+: A loop that alternates between receiving user input and
 drawing to the screen.
 
-*Hit testing*: Determining which element or accessibility tree node
+Hit testing
+
+: Determining which element or accessibility tree node
 is at a given pixel location on the screen.
 
-*Invalidation*: Marking some rendering state as no longer valid, because its
+Invalidation
+
+: Marking some rendering state as no longer valid, because its
 input dependencies have changed.
 
-*Layout*: The phase of a browser rendering pipeline that determines the
+Layout
+
+: The phase of a browser rendering pipeline that determines the
 size and position of elements in the DOM. 
 
-*Layout tree*: A second tree that mirrors the DOM, except that it represents
+Layout tree
+
+: A second tree that mirrors the DOM, except that it represents
 the output of the layout pipeline phase.
 
-*Paint*: The phase of a browser rendering pipeline that creates a display
+Paint
+
+: The phase of a browser rendering pipeline that creates a display
 list from the DOM.
 
-*Rendering pipeline*: The sequence of phases by which a browser draws
-a web page onto a computer screen.
+Raster
 
-*Raster*: The process of executing a display list by coloring the
+: The process of executing a display list by coloring the
 pixels of a surface.
 
-*Scroll*: adjusting the horizontal or vertical offset of a web page
+Rendering pipeline
+
+: The sequence of phases by which a browser draws
+a web page onto a computer screen.
+
+Scroll
+
+: adjusting the horizontal or vertical offset of a web page
 in response to user input, in order to see parts of it not currently visible.
 
-*Style*: The phase of a browser rendering pipeline that applies CSS rules to
+Style
+
+: The phase of a browser rendering pipeline that applies CSS rules to
 determine the visual appearance and behavior of elements in the DOM.
 Or, the set of CSS properties applied to an element after this phase.
 
-*Surface*: A buffer or texture within a GPU that represents a 2D array of
+Surface
+
+: A buffer or texture within a GPU that represents a 2D array of
 pixels.
 
-*Visual effect*: A CSS property that does not affect layout.
+Visual effect
+
+: A CSS property that does not affect layout.
 
-*Zoom*: Changing the ratio of CSS sizes to pixels in order to make
+Zoom
+
+: Changing the ratio of CSS sizes to pixels in order to make
 content on a web page larger or smaller.
 
 Computer technologies
 =====================
 
-*Assistive technology*: Computer software used to assist people in using
+Assistive technology
+
+: Computer software used to assist people in using
 the computer or web browser. The most common are screen readers.
 
-*CPU*: Central Processing Unit, the hardware component in a computer that
+CPU
+
+: Central Processing Unit, the hardware component in a computer that
 executes generic compute programs.
 
-*DukPy*: A JavaScript interpreter used in this book.
+DukPy
+
+: A JavaScript interpreter used in this book.
```

Comment:
Should we say "JavaScript engine" instead of "interpreter"? I believe it *is* an interpreter but probably not relevant. And "engine" is broader, encompassing a runtime, a garbage collector, and so on.